### PR TITLE
Fix airgap for fleet

### PR DESCRIPTION
**Pull request by StrongMonkey**
This commit add global registry setting and pass it to the fleet charts
that are installed by controller, since these settings can't be passed
from UI.

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/29143

File: pkg/controllers/dashboard/fleetcharts/controller.go [hunk 1]
```diff
@@ -55,9 +55,15 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 		return setting, err
 	}
 
+	systemGlobalRegistry := map[string]interface{}{
+		"cattle": map[string]interface{}{
+			"systemDefaultRegistry": settings.SystemDefaultRegistry.Get(),
```
Comment (ibuildthecloud) [inline pkg/controllers/dashboard/fleetcharts/controller.go:60]: This needs to be on L49 also

Comment (StrongMonkey) [reply pkg/controllers/dashboard/fleetcharts/controller.go:60]: added

**Review by cbron — APPROVED** [review]
I don't have wider context here but seems like it could work

**Review by ibuildthecloud — APPROVED** [review]

**Pull request merged**